master/pkg/actor/pool/pool.go

```diff
@@ -0,0 +1,171 @@
+package pool
+
+import (
+	"github.com/determined-ai/determined/master/pkg/actor"
+)
+
+// ActorPoolQueueFullError is returned by SubmitTask when the pool is so backed up that the queue
+// is full.
+type ActorPoolQueueFullError struct{}
```

Comment:
maybe just `QueueFullError` since it will be imported as `pool` and written `pool.QueueFullError`.